azerothcore/azerothcore-wotlk
# fix(DB): Creature Placements Southeast of Dolanaar

## Changes Proposed:
-  fixes xyzo spawnpoints and gives better estimate movement based on sniffs
-  respawns set to lowest observed regular spawn timers while alone in area

## Issues Addressed:

- Closes 

## SOURCE:

Sniffs uploaded to AC

## Tests Performed:

- checked DBError.log and checked area impacted by PR for mistakes


## How to Test the Changes:


1. Run SQL and check the creatures inserted, they are all listed in the deletion that starts on line 5

## Known Issues and TODO List:


- It was discovered that the core is adding 30 seconds to respawns, I do not know why but I suspect it's to prevent 0 second respawns and prolly needs an If statement somewhere rather than adding 30 seconds to every spawn.


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
